### PR TITLE
Added additional test scenarios to Feign Demo

### PR DESCRIPTION
**Pull request by vilmantas-r**

### CODE REVIEW

File: feign-demo/src/test/java/com/nortal/test/feign/demo/glue/TestPageStepDefs.java [hunk 1]
```diff
@@ -22,19 +22,98 @@
...
+            scenarioContext.putStepData(RESPONSE, responseEntity);
+        } catch (HttpClientErrorException ex) {
+            scenarioContext.putStepData(RESPONSE, ex);
+        }
+    }
+
+    @Then("user should get response with cat fact")
+    public void assertCatFact() {
+        ResponseEntity<CatApi.Fact> responseEntity = scenarioContext.getStepData(RESPONSE);
+        assertNotNull(responseEntity, "Response should not be null");
```
Comment (DonMelo) [inline feign-demo/src/test/java/com/nortal/test/feign/demo/glue/TestPageStepDefs.java]: Assertions are being duplicated, we could put them in a different method to clean up our code

File: feign-demo/src/test/java/com/nortal/test/feign/demo/glue/TestPageStepDefs.java [hunk 2]
```diff
@@ -22,19 +22,98 @@
...
+            scenarioContext.putStepData(RESPONSE, responseEntity);
+        } catch (HttpClientErrorException ex) {
+            scenarioContext.putStepData(RESPONSE, ex);
+        }
+    }
+
+    @Then("user should get response with a list of cat facts")
+    public void assertCatFacts() {
+        ResponseEntity<CatApi.FactsList> responseEntity = scenarioContext.getStepData(RESPONSE);
+        assertNotNull(responseEntity, "Response should not be null");
```
Comment (DonMelo) [inline feign-demo/src/test/java/com/nortal/test/feign/demo/glue/TestPageStepDefs.java]: Assertions are being duplicated, we could put them in a different method to clean up our code

File: feign-demo/src/test/java/com/nortal/test/feign/demo/glue/TestPageStepDefs.java [hunk 3]
```diff
@@ -22,19 +22,98 @@
...
+            scenarioContext.putStepData(RESPONSE, responseEntity);
+        } catch (HttpClientErrorException ex) {
+            scenarioContext.putStepData(RESPONSE, ex);
+        }
+    }
 
+    @Then("user should get response with a list of cat breeds")
+    public void assertBreeds() {
+        ResponseEntity<CatApi.BreedsList> responseEntity = scenarioContext.getStepData(RESPONSE);
+        assertNotNull(responseEntity, "Response should not be null");
```
Comment (DonMelo) [inline feign-demo/src/test/java/com/nortal/test/feign/demo/glue/TestPageStepDefs.java]: Assertions are being duplicated, we could put them in a different method to clean up our code

File: feign-demo/src/test/java/com/nortal/test/feign/demo/glue/TestPageStepDefs.java (hunk 1 above)
Comment (vilmantas-r) [reply feign-demo/src/test/java/com/nortal/test/feign/demo/glue/TestPageStepDefs.java]: Fixed, added assertResponse method

**Review by DonMelo — APPROVED** [review]

**Pull request merged**